common/focuses/00_test_focuses.txt

```diff
@@ -1,115 +0,0 @@
-﻿#test load
```

Comment:
I compared the backgrounds to the vanilla.

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/111283442-93cf1c00-8658-11eb-8b76-c8b421400fc2.png)
![image](https://user-images.githubusercontent.com/46384992/111283465-992c6680-8658-11eb-86ee-9b7f41e94fd5.png)

</details>

So I would blur it a bit more and make it less saturated to fit the window and focus on perks themselves.

Reply:
Bruh, but you yourself said that it looks good..

https://discord.com/channels/317351018050355200/751109648869752954/804643551224659988

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/111285236-fa9d0700-8651-11eb-8c66-40c09b0299f0.png)

![image](https://user-images.githubusercontent.com/46576860/111285124-db9e7500-8651-11eb-9f7d-3fdcffd4960f.png)

</details>

Reply:
> Bruh, but you yourself said that it looks good..

I haven't seen the interface in-game until now.

Reply:
Tell me if you want it even more blurry and less saturated

Reply:
> 
> 
> Tell me if you want it even more blurry and less saturated

Any more blurry and less saturated and you wouldn't tell there were color there at all. They look like vanilla ones now.